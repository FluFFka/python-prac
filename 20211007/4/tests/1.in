"x**2-sin(x)+log(x)*exp(x)", "-789456486534798465198465", "x*8+y-x*sin(y)"
123
