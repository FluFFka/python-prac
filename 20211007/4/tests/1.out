9.649503307628529e+54
